knowledge-base/chart-area-gradient-color.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Apply gradient effect on Area Chart
+description: How to apply gradient as color for Area Chart series?
+type: how-to
+page_title: Apply gradient effect on Area Chart
+slug: chart-area-gradient-color
+position: 
+tags: chart, area, gradient, color
+ticketid: 1591373
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Chart for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+How can I apply a gradient color effect on the Telerik UI for Blazor Area Chart?
+
+## Solution
+
+1. Declare an external gradient.
+1. Use the [Color](https://docs.telerik.com/blazor-ui/components/chart/types/area#color) configuration of the Chart series to set the external gradient. 
+
+>caption How to apply gradient effect on Area Chart.
+
+````CSHTML
+@*Gradient*@
+<div style="height: 0">
+    <svg xmlns="https://www.w3.org/2000/svg" version="1.1" width="0" height="0" style="visibility: hidden">
+        <defs>
+            <linearGradient id="svg-gradient" x1="0%" y1="0%" x2="0%" y2="100%">
+                <stop offset="0%" style="stop-color:#3bafda; stop-opacity:0.95" />
+                <stop offset="100%" style="stop-color:#3bafda; stop-opacity:0.05" />
+            </linearGradient>
+        </defs>
+    </svg>
+</div>
+
+
+@*Chart*@
+<TelerikChart Class="chart">
```

Comment:
Do we need the `Class`? Try to remove anything that is not relevant to the example (except for required real-world settings that may force developers to add them back to make the scenario meaningful).

```suggestion
<TelerikChart>
```